PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/jobs/framework/JobViewPanel.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+package com.microsoft.azure.hdinsight.jobs.framework
+
+import com.intellij.ui.jcef.JBCefApp
+import com.intellij.ui.jcef.JBCefBrowser
+import com.microsoft.azure.hdinsight.spark.jobs.JobViewHttpServer
+import java.awt.BorderLayout
+import javax.swing.JComponent
+import javax.swing.JPanel
+
+class JobViewPanel(private val rootPath: String, private val clusterName: String) {
+    private val QUERY_TEMPLATE = "?clusterName=%s&port=%s&engineType=javafx"
```

Comment:
Just curious, why the engineType is javafx?

Reply:
Not sure about the real function of engineType. But I found the engine type is also used in Eclipse code, so I just leave it here.
![image](https://user-images.githubusercontent.com/32627233/110620712-4f64fb80-81d4-11eb-9a75-63bd729f67c0.png)

Reply:
Maybe it's because the previous engine is just javafx and there are some workaround for it? How about remove it and have a test?

Reply:
Just as you searched in code, nothing found about the engineType reference, we can change the engineType here to `jcef`

Reply:
Removed the `javaFx` related context in the new iteration.